### PR TITLE
MK-1715

### PR DESCRIPTION
**Pull request by smirfolio**
Search: Add a scroll up of the vocabularies-terms panel

### CODE REVIEW

**Comment by ymarcon** [comment]
Are you sure that you cannot use bootstrap variables?

**Comment by smirfolio** [comment]
It's an custom widget, that is not available in bootstrap, so i'm not sure that bootstrap variables exist

**Comment by ymarcon** [comment]
Despite it is a custom widget, you must make sure that the colors fit with the the (bootrap) theme and I am pretty sure that there are color variables that could be used.

File: less/obiba.less [hunk 1]
```diff
@@ -604,4 +604,29 @@ fieldset legend.panel-heading {
...
+#back-to-top {
+  position: fixed;
+  bottom: 40px;
+  right: 40px;
+  z-index: 9999;
+  width: 32px;
+  height: 32px;
+  text-align: center;
+  line-height: 30px;
+  background: #f5f5f5;
```
Comment (ymarcon) [inline less/obiba.less]: Find a bootstrap color variable for this.

File: less/obiba.less [hunk 2]
```diff
@@ -604,4 +604,29 @@ fieldset legend.panel-heading {
...
+  color: #444;
+  cursor: pointer;
+  border: 0;
+  border-radius: 2px;
+  text-decoration: none;
+  transition: opacity 0.2s ease-out;
+  opacity: 0;
+}
+#back-to-top:hover {
+  background: #e9ebec;
```
Comment (ymarcon) [inline less/obiba.less]: Find a bootstrap color variable for this.

File: less/obiba.less [hunk 3]
```diff
@@ -604,4 +604,29 @@ fieldset legend.panel-heading {
...
+  position: fixed;
+  bottom: 40px;
+  right: 40px;
+  z-index: 9999;
+  width: 32px;
+  height: 32px;
+  text-align: center;
+  line-height: 30px;
+  background: #f5f5f5;
+  color: #444;
```
Comment (ymarcon) [inline less/obiba.less]: Find a bootstrap color variable for this.

File: less/obiba.less (hunk 2 above)
Comment (smirfolio) [reply less/obiba.less]: I figure out what you mean, and i have added a btn class to the widget : 
https://github.com/obiba/bootstrap-drupal7/pull/60/commits/96121b754f20fd863d05dd5115d6d218b31699b3

https://github.com/obiba/mica-drupal7/pull/1515/commits/c952c4dbc9383bd3634827f8456f7538f81c976f#diff-b3a24b35451b73a97516ae75aafe6d9fR17

**Pull request merged**